transmitter.js

```diff
@@ -1,31 +1,28 @@
 let transmissionFailureCount = 0;
 
-function networkTransmitStub(celcius) {
-    console.log(`Temperature to transmit: ${celcius} celcius`);
+function networkTransmitStub(celsius) {

```

Comment:
You have separated test-code into transmitter test js, but left the stub in here.
The stub is test code - it's used only in testing. Move the stub out of this file into the test. 
Use dependency injection to break the circular dependency, as discussed in the meeting